model/installation_request.go

```diff
@@ -81,9 +82,45 @@ func (request *CreateInstallationRequest) Validate() error {
 		return errors.Wrap(err, "invalid env var settings")
 	}
 
+	return checkSpaces(request)
+}
+
+func checkSpaces(request *CreateInstallationRequest) error {
+	if hasWhiteSpace(request.OwnerID) != -1 {
+		return errors.Errorf("cannot have spaces in owner field. Owner=%s", request.OwnerID)
+	}
```

Comment:
I think technically it's fine to have spaces in the OwnerID field. It's just metadata.

Reply:
right